src/configuration/LimitsSettings.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { SyntheticEvent } from 'react';
+
+import {
+  DataSourcePluginOptionsEditorProps,
+  SelectableValue,
+} from '@grafana/data';
+import {
+  Button,
+  EventsWithValidation,
+  LegacyForms,
+  regexValidation
+} from '@grafana/ui';
+
+import { LimitMetrics, PromOptions } from '../types';
+
+import { getValueFromEventItem } from "./PromSettings";
+
+const { Input, FormField } = LegacyForms;
+
+const limitsSettingsValidationEvents = {
+  [EventsWithValidation.onBlur]: [
+    regexValidation(
+      /^$|^\d+$/,
+      'Value is not valid, you can use number'
+    ),
+  ],
+};
+
+const limitFields = [
```

Comment:
I think would be nice to use descriptions from https://github.com/VictoriaMetrics/VictoriaMetrics/blob/2e04ddbd327962f1b355c9381be9b97915169844/README.md?plain=1#L1459-L1461

Reply:
updated